config/fields/mixins/picker.php

```diff
@@ -98,20 +107,27 @@
 		'getIdFromArray' => function (array $array) {
 			return $array['uuid'] ?? $array['id'] ?? null;
 		},
+		'itemsFromRequest' => function () {
+			$ids    = $this->kirby()->request()->get('items', '');
+			$ids    = Str::split($ids);
+			$models = $this->toModels($ids);
+			return $this->toItems($models);
+		},
+
 		'toFormValues' => function ($value = null) {
-			$items = [];
+			$ids = [];
 
 			foreach (Data::decode($value, 'yaml') as $id) {
 				if (is_array($id) === true) {
 					$id = $this->getIdFromArray($id);
 				}
 
 				if ($id !== null && ($model = $this->toModel($id))) {
-					$items[] = $this->toItem($model);
+					$ids[] = $model->id();
 				}
 			}
 
-			return $items;
+			return $ids;
```

Comment:
`Review:` Does it make sense that we always use IDs here and not UUIDs? Since we use `->toModel()->{$store)` in the `save` callback, it should work consistently to always just communicate with the IDs. But wondering if conceptually we should also use `$uuid?? $id` here.